clients/admin-ui/src/features/common/table/FidesTable.tsx

```diff
@@ -133,7 +134,7 @@ export const FidesTable = <T extends FidesObject>({
                     ? { backgroundColor: "gray.50", cursor: "pointer" }
                     : undefined
                 }
-                data-testid={`row-${rowName}`}
+                data-testid={`row-${rowName ?? rowId}`}
```

Comment:
privacy experiences don't have a `name` field, so rely on `id` for the test id instead